templates/relnotes.md

```diff
@@ -22,6 +22,8 @@ Platform Support
 Refer to Rust's [platform support page][platform-support-doc]
 for more information on Rust's tiered platform support.
 
+[platform-support-doc]: https://doc.rust-lang.org/rustc/platform-support.html
```

Comment:
This is semi-intentionally omitted since in the eventual RELEASES.md there's already such a link... I guess having duplicates is probably mostly harmless?

Reply:
Ah... that's what I was missing. It didn't occur to me that it'd be on the bottom in the actual combined markdown.